pages/emotion/[id]/edit.js

```diff
@@ -20,10 +20,15 @@ export default function EditPage({ objects, onUpdateEmotion }) {
           onUpdateEmotion({ ...data, id });
           router.push(`/emotion/${id}`);
         }}
-        onCancel={() => {
+      />
+      <button
+        type="button"
+        onClick={() => {
           router.push(`/emotion/${id}`);
         }}
-      />
+      >
+        Cancel
+      </button>
```

Comment:
Replace the button with Next.js' `<Link>` component instead of using imperative routing (`router.push`).

Using `<Link>` is preferable because it enables prefetching, optimizing page load performance,  and it aligns with Next.js best practices for handling navigation declaratively.

🤓 This approach also simplifies the code and reduces the risk of potential side effects from manual routing.